lib/jsonite/lets_proxy.rb

```diff
@@ -0,0 +1,23 @@
+class Jsonite
+  class LetsProxy < BasicObject
+
+    undef_method :==, :!, :!=
+
+    def initialize object, context, lets = {}
+      @__object, @__context, @__lets = object, context, lets
```

Comment:
Actually... realizing that only the methods there have the prefix/suffix. We can probably just use the simple `_` prefix and roll with it. Don't think it's dangerous since even `instance_variable_get` proxies through to the object.